src/synapso_cli/commands/query.py

```
@@ -1,8 +1,18 @@
 def cmd_query(query: str):
     """Execute a query against a cortex."""
-    query_manager = QueryManager()
-    result = query_manager.query(query)
-    return result
+    rest_client = get_rest_client()
+    try:
+        response = rest_client.query(query)
+        typer.echo(response)
+    except SynapsoRestClientError as e:
+        typer.echo(f"Synapso REST client error: {e}", err=True)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)